jwst/datamodels/container.py

```diff
@@ -479,12 +483,12 @@ def ind_asn_type(self, asn_exptype):
         Parameters
         ----------
         asn_exptype : str
-            Exposure type as defined in an association, e.g. "science".
+            Exposure type as defined in an association, e.g., "science".
 
         Returns
         -------
         ind : list
-            Indices of models in ModelContainer._models matching ``asn_exptype``.
+            Indices of models in ``ModelContainer._models`` matching ``asn_exptype``.
```

Comment:
```suggestion
            Indices of models in the container matching ``asn_exptype``.
```
Seems weird to explicitly mention a hidden attribute here